pkgs/development/python-modules/salib/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
```

```suggestion
  buildPythonPackage,
  ...
```
see also https://github.com/NixOS/nixpkgs/pull/344042#discussion_r1772402543